src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -342,10 +376,10 @@ public function setTheme($theme)
             'bullist', 'numlist', 'outdent', 'indent',
         ],
         2 => [
-            'formatselect', '|',
-            'paste', 'pastetext', '|',
+            'blocks', '|',
+            'pastetext', '|',
             'table', 'sslink', 'unlink', '|',
-            'code'
+            'code visualblocks'
```

Comment:
Should it be two separate command `'code', 'visualblocks'`?

Reply:
Both work - but yeah it should probably be separated just so it's consistent with the rest of the items here.

Reply:
done